fvtest/compilertriltest/ShiftAndRotateTest.cpp

```diff
@@ -56,6 +56,34 @@ template <typename T> static T rotate(T a, int32_t b)
    return a;
    }
 
+template <typename Left, typename Right> static Left shift_left(Left left, Right right)
```

Comment:
Since, the "right" argument to a shift opcode must always be `Int32`, can you get rid of the `Right` template parameter and hard code `int32_t ` instead? This will make it consistent with the implementation of `rotate`.

Reply:
Changed